test/test_IO.py

```diff
@@ -53,7 +54,7 @@ def test_overwrite_existing_folder(tmpdir):
             (test_dir / "file.txt").touch()
 
             assert not _directory_is_empty(test_dir)
-            remove_and_create_folder(test_dir)
+            remove_and_create_folder(test_dir, force=True)
             assert _directory_is_empty(test_dir)
```

Comment:
Can you add a test case ensuring things work as expected when `force=False`?

Reply:
I added a few lines with proper assert.